modules/configIO.lua

```diff
@@ -1,45 +1,65 @@
+-- This files generates the configIO & merge it with CUSTOM channels defined in 'vendor.configIO'
+
 local configIO = {}
+local vendorIO = require("vendor.configIO")
 
 function configIO.get()
   local now = os.time(os.date("!*t"))
-  if configIOCache == nil or now - configIOCache_ts > 10 then
+  if globalConfigIOCache == nil or now - globalConfigIOCache_ts > 10 then
     local result = Keystore.get({ key = "config_io" })
-    if result ~= nil and result.value ~= nil then
+    if result ~= nil and result.value ~= nil and result.value ~= "" then
       -- Following is a VM global value cached on hot VM
-      configIOCache, err = json.parse(result.value)
+      globalConfigIOCache, err = json.parse(result.value)
       if err ~= nil then
-        print("The config_io parse error", err)
+        log.error("'config_io' parsing error", err)
       else
-        configIOCache_ts = now
+        globalConfigIOCache_ts = now -- global
```

Comment:
where is this `global` defined?

Reply:
oh sorry, `--` means comments, i though is an array operator.

Reply:
:) but I added in the name so its not necessary.